_plugins/yaml_content_disposition.rb

```diff
@@ -0,0 +1,16 @@
+require 'webrick'
+
+module Jekyll
+  module Commands
+    class Serve
+      class Servlet < WEBrick::HTTPServlet::FileHandler
+        prepend(Module.new do
+          def do_GET(req, res)
+            res.header.merge!('Content-Disposition' => 'attachment') if req.path.end_with?('.yml')
+            super
+          end
+        end)
```

Comment:
not a super big fan of `end)` but this seems like the simplest overall, and we aren't going to touch this file that often

other ideas:

too sneaky:
```ruby
Module.new do
  # ...
end.tap { |mod| prepend mod }
```
more verbase
```ruby
attachment_mod = Module.new
  # ...
end

prepend attachment_mod
```

Reply:
Yeah, and for context this is the form we've taken with other plugins which are monkeypatching Jekyll internals ([example](https://github.com/18F/identity-handbook/blob/main/_plugins/content_typography.rb)).